doc/dune-files.rst

```diff
@@ -1225,11 +1210,12 @@ To use Menhir in a Dune project, the language version should be selected in the
 
   (using menhir 2.0)
 
-This will enable support for Menhir stanzas in the current project. If the
 language version is absent, Dune will automatically add this line with the
+This will enable support for Menhir stanzas in the current project. If the
```

Comment:
“If the latest Menhir version once a Menhir stanza is used anywhere.” 
This sentence isn’t clear.

Reply:
Looks like the lines swapped!